### ISSUE TITLE
Pie chart fill color is pink

### ISSUE THREAD

**Issue by tedberg** [dc-js/dc.js#1117]
![screen shot 2016-03-11 at 2 17 51 pm](https://cloud.githubusercontent.com/assets/815032/13717115/2f500f68-e794-11e5-8a52-c84180c70885.png)

Go to demo page: https://dc-js.github.io/dc.js/

Use selectors to make pie charts empty.  The default style has white text on a pink background.

**Comment by gordonwoodhull** [comment]
Yes, it is pink. It is a special case that is styled in CSS.

https://github.com/dc-js/dc.js/blob/develop/dc.css#L30

If this color offends you, you can specify your own color in a rule for `.dc-chart .empty-chart .pie-slice path`

**Comment by tedberg** [comment]
Yes, this is what I do to change it to a shade of light gray.  I match the color used for excluded slices when another slice is selected.  

I would think most people would equate null to a color like that.  Bug was just a suggestion to default to grey.

**Comment by gordonwoodhull** [comment]
I see. Sure, the "excluded color" may be a better default here. Thanks for clarifying.
